layouts/_default/baseof.html

```diff
@@ -13,8 +13,8 @@
         <link rel="stylesheet" href="{{ .RelPermalink }}">
       {{ end }}
     {{ end }}
-    <script src="{{ "/assets/scripts/jquery.js" | absURL }}"></script>
-    <script src="{{ "/assets/scripts/bootstrap.js" | absURL }}"></script>
+    <script src="{{ "assets/scripts/jquery.js" | absURL }}"></script>
+    <script src="{{ "assets/scripts/bootstrap.js" | absURL }}"></script>
     {{ if $.Site.Params.lazyLoad }}
```

Comment:
@WiaanB Does this also work if you are within a sub-directory? E.g https://localhost:8081/docs/span-handbook/span-overview/#spans-vision-and-mission

Reply:
Yes, it does